src/main/java/no/nav/veilarbvedtaksstotte/controller/AdminController.java

```diff
@@ -55,7 +55,19 @@ public void fjernUtrulling(@PathVariable EnhetId enhetId) {
     @PostMapping("/republiser/siste-14a-vedtak-fra-vedtaksstotte")
     public String republiserSiste14aVedtakFraVedtaksstotte() {
```

Comment:
Hvorfor finnes det to muligheter for republisering? Er det fortsatt relevant?

Reply:
Godt poeng det, kanskje like greit å fjerne den gamle med en gang.